phovea.js

```diff
@@ -26,20 +26,14 @@ module.exports = function (registry) {
     'name': 'TimelineView'
   });
 
-  registry.push('malevoView', 'DetailView', function () {
-    return System.import('./src/detail_view/DetailView');
-  }, {
-    'name': 'DetailView'
-  });
-
   registry.push('malevoView', 'ToolbarView', function () {
     return System.import('./src/toolbar/ToolbarView');
   }, {
     'name': 'ToolbarView'
   });
 
   registry.push('malevoView', 'DetailChartTab', function () {
```

Comment:
Since no other application is using this view it's save to rename the identifier. Please rename the id accordingly to the class.